### DISCUSSION TITLE
hide in github like passwords?

### DISCUSSION THREAD

**Discussion by sainived151** [community/community#140315] [New to GitHub]
how to hide my long strings in github like passwords?

**Comment by resort3779** [comment]
you can use .env files and add the file to gitignore

**Reply by generalsunion** [reply]
#140315

**Comment by abdulrehmanwaseem** [comment]
**Create Environment Files in your root directory (.env) and also add that name to .gitignore file for not leaking your env variables to publi**c

**Reply by Mmohmd** [reply]
#

**Reply by Mmohmd** [reply]
>
